### ISSUE TITLE
Process finished with exit code 132 (interrupted by signal 4: SIGILL)

### ISSUE THREAD

**Issue by eam2000** [tensorflow/tensorflow#47799]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS Big Sur 11.2.1 Apple M1
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 2.4.1
- Python version: 3.8

**Describe the current behavior**
After running my script I receive the following error: "Process finished with exit code 132 (interrupted by signal 4: SIGILL)"

**Describe the expected behavior**
I expect the script to run entirely

**Standalone code to reproduce the issue**

import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

training_dataset = pd.read_csv('/Users/--/Downloads/Archive/AAPL.csv')
print(training_dataset)
training_set = training_dataset.iloc[:,1:-1].values
print(training_set)
from sklearn.preprocessing import MinMaxScaler

sc = MinMaxScaler(feature_range = (0, 1))
training_set_scaled = sc.fit_transform(training_set)
print(training_set_scaled)

X_train = []
Y_train = []

for i in range(252,len(training_set)):
    X_train.append(training_set_scaled[i - 252:i,0])
    Y_train.append(training_set_scaled[i,0])

X_train, Y_train = np.array(X_train), np.array(Y_train)

X_train = np.reshape(X_train, (X_train.shape[0], X_train.shape[1], 1))

from keras.models import Sequential
from keras.layers import Dense
from keras.layers import LSTM
from keras.layers import Dropout

model = Sequential()

model.add(LSTM(units = 50, return_sequences = True, input_shape = (X_train.shape[1], 1)))

model.add(Dropout(0.2))

model.add(model.add(LSTM(units = 50, return_sequences = True)))

model.add(Dropout(0.2))

model.add(model.add(LSTM(units = 50, return_sequences = True)))

model.add(Dropout(0.2))

model.add(model.add(LSTM(units = 50, return_sequences = True)))

model.add(Dropout(0.2))

model.add(Dense(units = 1))

model.compile(optimizer = 'adam', loss = 'mean_squared_error')

model.fit(X_train, Y_train, epochs = 100, batch_size = 32)

**Comment by Saduf2019** [comment]
@eam2000 
I ran the code shared and find some dependencies missing, please find the [gist here](https://colab.research.google.com/gist/Saduf2019/524acfb5edf6f18c9f4c14e4bcd59ca0/untitled561.ipynb)
